### PR TITLE
WL-3107 Stop our administrator email sounding gibbon like.

### PR DESCRIPTION
**Pull request by buckett**
This has an additional commit fixing references to the Student Enrollment System which doesn't have a Jira, but is small and depends on the WL-3107 work which is why it's in here for simplicity.

### CODE REVIEW

**Comment by nicholaswilson100** [comment]
Are you saying our adminstrators sound like gibbons? Can I quote you?

File: impl/src/main/java/uk/ac/ox/oucs/vle/proxy/SakaiProxyImpl.java [hunk 1]
```diff
@@ -391,11 +391,11 @@ public String getConfirmUrl(String signupId, String placementId) {
 		if (null == signupId) {
 			return getUrl("/static/pending.jsp", placementId);
 		}
-		return getUrl("/static/pending.jsp#"+ signupId, placementId);
+		return getUrl("/static/pending.jsp#" + signupId, placementId);
 	}
```
Comment (nicholaswilson100) [inline impl/src/main/java/uk/ac/ox/oucs/vle/proxy/SakaiProxyImpl.java]: Apart from the de-gibboning text changes, did you mean to put in spaces around the place like here?

**Comment by buckett** [comment]
That's what's in the Jira.

File: impl/src/main/java/uk/ac/ox/oucs/vle/proxy/SakaiProxyImpl.java (hunk 1 above)
Comment (buckett) [inline impl/src/main/java/uk/ac/ox/oucs/vle/proxy/SakaiProxyImpl.java]: IntelliJ "being good", I'll tidy up.

**Comment by buckett** [comment]
Fixed intellij tidying.

**Pull request merged**